src/borg/helpers/checks.py

```diff
@@ -8,9 +9,14 @@ class PythonLibcTooOld(Error):
 
 
 def check_python():
-    required_funcs = {os.stat, os.utime, os.chown}
+    if sys.platform.startswith(('win32', )):
```

Comment:
I agree… 4 Flags... IsWin, IsLinux, IsBsd, IsDarwin instead of this massive abuse or try except** we should use this flags step by step. The real positive effect is that we reveal the real usage of the code.
(Also as a basis for organizing this code later)

** Did none use a debugger? I stopped using the debugger after two trys, because of this many exceptions making it impossible.

Reply:
Done